src/style.css

```diff
@@ -1,3 +1,89 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the font 'Roboto' is loaded correctly in your HTML file. If it's not loading, verify the link to Google Fonts in your HTML file.